tests/Meilisearch.Tests/KeyTests.cs

```diff
@@ -0,0 +1,119 @@
+namespace Meilisearch.Tests
+{
+    using System.Linq;
+    using System.Threading.Tasks;
+    using FluentAssertions;
+    using Xunit;
+
+    [Collection("Sequential")]
+    public class KeyTests : IAsyncLifetime
+    {
+        private IndexFixture fixture;
+        private MeilisearchClient client;
+
+        public KeyTests(IndexFixture fixture)
+        {
+            this.fixture = fixture;
+            this.client = fixture.DefaultClient;
+        }
+
+        public async Task InitializeAsync()
+        {
+            await this.fixture.DeleteAllIndexes(); // Test context cleaned for each [Fact]
+        }
+
+        public Task DisposeAsync() => Task.CompletedTask;
+
+        [Fact]
+        public async Task GetAllKeys()
+        {
+            var keyResponse = await this.client.GetKeysAsync();
+            var keys = keyResponse.Results;
+            keys.Count().Should().BeGreaterOrEqualTo(2);
```

Comment:
(just adding a line before the return)
```suggestion

            keys.Count().Should().BeGreaterOrEqualTo(2);
```